types/binary-parser/index.d.ts

```diff
@@ -115,7 +115,7 @@ export interface Parser<O extends object | undefined = undefined> {
 
     seek(length: number): Parser<O>;
 
-    endianess(endianess: Parser.Endianness): Parser<O>; /* [sic] */
+    endianness(endianness: Parser.Endianness): Parser<O>; /* [sic] */
```

Comment:
It looks like it supports both `endianess` and `endianness`?

Reply:
Before this PR a mix of both "versions" of this word existed here.